docs/getstarted/userinterface.md

```diff
@@ -279,6 +279,36 @@ You can Drag and Drop editor groups on the workbench, move individual Tabs betwe
 
 >**Note:** VS Code uses editor groups whether or not you have enabled Tabs.  Without Tabs, editor groups are a stack of your open items with the most recently selected item visible in the editor pane.
 
+### Locked editor groups
+
+By default, files will always open in the active editor group. Locked editor groups help to avoid this. The `workbench.editor.autoLockGroups` setting allows you to select editors that should lock a group automatically when they open. This only applies when the editor is the first to open in an otherwise empty or new group.
+
+```json
+"workbench.editor.autoLockGroups": {
+  "default": false,
+  "terminalEditor": true
+}
+```
+
+Terminals are configured by default to cause a new group to lock automatically. A lock icon in the action toolbar (top right) indicates a locked group. The locked state of an editor group is persisted and restored across restarts. You can lock empty groups for a more stable editor layout.
```

Comment:
Perhaps the final sentence here about locking empty groups would make more sense further down after the concept of manual locking has been covered. Also maybe reference the setting that controls whether a group closes automatically when it becomes empty.

Reply:
```suggestion
Terminals are configured by default to cause a new group to lock automatically. A lock icon in the action toolbar (top right) indicates a locked group. The locked state of an editor group is persisted and restored across restarts.
```

Moving the sentence.